test/unit/location_test.rb

```diff
@@ -311,7 +311,7 @@ class LocationTest < ActiveSupport::TestCase
 
   test "location name can't be too big to create lookup value matcher over 255 characters" do
     parent = FactoryGirl.create(:location)
-    min_lookupvalue_length = "location=".length + parent.title.length + 1
+    min_lookupvalue_length = "location=".length + parent.title.length + 1 # rubocop:disable Performance/FixedSize
```

Comment:
Hmm how about putting the whole /test directory under 'Exclude' for this cop? Doing this every time seems excessive